Drawsana/Helpers/ImmediatePanGestureRecognizer.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  ImmediatePanGestureRecognizer.swift
+//  Drawsana
+//
+//  Created by Steve Landey on 8/14/18.
+//  Copyright © 2018 Asana. All rights reserved.
+//
+
+import UIKit
+import UIKit.UIGestureRecognizerSubclass
+
+class ImmediatePanGestureRecognizer: UIGestureRecognizer {
```

Comment:
Might be worth adding a class header comment to describe what this class is used for